tensorflow/lite/delegates/webnn/webnn_delegate.cc

```diff
@@ -3096,6 +3096,11 @@ TfLiteIntArray* Delegate::PrepareOpsToDelegate(TfLiteContext* context) {
             &nodes_to_delegate->data[0]);
 #endif
 
+  TFLITE_LOG_PROD_ONCE(tflite::TFLITE_LOG_INFO,
+                       "TensorFlow Lite WebNN delegate capability:"
+                       " number of nodes in the graph: %d,"
+                       " number of nodes supported by WebNN: %d.",
+                       execution_plan->size, nodes_to_delegate->size);
```

Comment:
What would the info look like if there are multiple subgraphs delegated by WebNN?

Reply:
```
tflite_web_api_cc_simd.js:9 INFO: TensorFlow Lite WebNN delegate capability: number of nodes in the graph: 164, 
number of nodes supported by WebNN: 153.
p
```